src/parser/jobs/gnb/modules/Ammo.tsx

```diff
@@ -55,12 +55,45 @@ export class Ammo extends CoreGauge {
 	override initialise() {
 		super.initialise()
 
-		if (this.parser.patch.before('7.1')) {
-			DoubleDownCost = 2
+		if (this.parser.patch.before('7.1') || this.parser.patch.after('7.3')) {
+			this.ammoModifiers.set(this.data.actions.DOUBLE_DOWN.id, {action: -2})
```

Comment:
Most of the other cases that I can recall for having patch-wise const changes did it by making two different const variables for the different values, and conditional logic for assigning the functional variable to those, rather than an assign-and-conditional-modify like this. It still works, and either way is kinda clunky, so I'll leave it up to you if you want to change it.

Reply:
I'm not sure if the ariable names are great, but updated to consts.
const DOUBLE_DOWN_COST_7_2_AND_7_3 = 1
const DOUBLE_DOWN_PRE_7_1 = 2 // also 7.4 and beyond!